src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkSpec.groovy

```diff
@@ -651,6 +654,50 @@ class LinkSpec extends HealthCheckSpecification {
         getIsl().srcSwitch.dpId | getIsl().srcPort | getIsl().dstSwitch.dpId | null      | "dst_port"
     }
 
+    @Tidy
+    @Tags(HARDWARE)
+    def "An ISL between noviflow switches doesn't go down when one switch is DEACTIVATED"(){
```

Comment:
Remove 'noviflow' from everywhere.
1. It is not because the switch is noviflow, it is because we use round trip latency to determine the isl state
2. When looking for switch, look at the switch features in order to determine whether the switch supports round trip latency (it can be an OVS switch in the future). The switch feature name which is used by system now is NOVIFLOW_COPY_FIELD

Reply:
fixed